packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -87,15 +90,46 @@ export class HorizontalBarChart extends FASTElement {
             color: point.color,
           });
         }
-      });
-    });
+      }
+    }
 
     // Convert the map values back to an array
     this.uniqueLegends = Array.from(uniqueLegendsMap.values());
   }
 
+  private _hydrateData() {
+    this.data!.forEach(({ chartTitle, chartData }) => {
+      if (chartData!.length === 1) {
+        const pointData = chartData![0];
+        const newEntry = {
+          legend: '',
+          data: pointData.total! - pointData.data! > 0 ? pointData.total! - pointData.data! : 0,
+          y: pointData.total!,
+          color: '#edebe9',
+          placeholder: true,
+        };
+        chartData!.push(newEntry);
+      }
+    });
+  }
+
+  private calculateBarSpacing(): number {
+    //todo: replace 650 with width of svg or div.
+    const svgWidth = 650;
```

Comment:
why hardcoding the width

Reply:
This I have added a TODO, I am working on it and will take this in next PR.